{{cookiecutter.project_slug}}/bitbucket-pipelines.yml

```
@@ -0,0 +1,61 @@
+image: python:3.10
+
+definitions:
+  services:
+    docker:
+      memory: 3072
+    postgres:
+      image: postgres:14
+      environment:
+        POSTGRES_DB: {{cookiecutter.project_slug}}
+        POSTGRES_USER: {{cookiecutter.project_slug}}
+        POSTGRES_PASSWORD: CHANGEME
+    redis:
+      image: redis:stable
+
+  steps:
+    - step: &backend-tests
+        name: Backend Tests
+        caches:
+          - pip
+        script:
+          - curl https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - && echo "deb http://apt.postgresql.org/pub/repos/apt buster-pgdg main" > /etc/apt/sources.list.d/pgdg.list
+          - apt-get update
+          - apt-get install --no-install-recommends -y build-essential ca-certificates curl gettext gnupg libpq-dev libffi-dev postgresql-client-12 postgresql-client-common
```

Let's use the postgresql-client meta package. This will install the latest supported version.
```suggestion
          - apt-get install --no-install-recommends -y build-essential ca-certificates curl gettext gnupg libpq-dev libffi-dev postgresql-client postgresql-client-common
```